Package.swift

```diff
@@ -67,6 +75,7 @@ let package = Package(
     .target(
       name: "PovioKitUIKit",
       dependencies: [
+        "Kingfisher",
```

Comment:
Not sure why this is needed if this subpackage already includes `PovioKitCore`?

Reply:
[Redundant](https://github.com/povio/PovioKit/pull/356/commits/fd642483f4cb65cdbb467f8be60dbd8707d3a067) yes.